rachael-t/whats-cookin
# Feature/test clean up

#### What's this PR do?

- Cleans up test files so that the data being used for instantiated objects is being pulled from a file instead of manually written into the test file. By doing so, this cleans up the test file for better readability. 

- Additionally, we noticed that while the user.searchRecipes function was tested and working, we never implemented the functionality on the DOM. Updated the scripts.js file so that this now works on the browser/UI

#### Where should the reviewer start?

- Reviewing the test files

#### How should this be manually tested?

- Running npm test tests to double check that all tests are still passing

- Logging in as a user, selecting recipes to save to favorites or to cook later, then searching "my recipes" by a name of a recipe or ingredient.

#### Any background context you want to provide?

- n/a

#### What are the relevant tickets?

- closes #6 

#### Screenshots (if appropriate)

- n/a

#### Questions:

- n/a
